specs/commandline/verify.md

```diff
@@ -2,7 +2,21 @@
 
 ## Description
 
-Use `notation verify` command to verify signatures on an artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. The digest of the supplied artifact is returned upon successful verification. It is recommended that this digest reference be used to pull the artifact subsequently, as registry tags may be mutable, and a tag reference can point to a different artifact that what was verified.
+Use `notation verify` command to verify signatures associated with the artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. Always verify the artifact using digest(`@sha256:...`) rather than a tag(`:latest`) because tags are mutable and a tag reference can point to a different artifact than that was verified.
```

Comment:
That's not true.

It is nature to verify a tagged manifest. Since the verification will return the digest of the verified manifest. Customers can then pull by digest.

Here's the workflow to verify and pull an image `example.contoso.io/app:v1`.
1. `notary verify example.contoso.io/app:v1`
2. If the above verification is successful, extract the digest from the output.
3. Use the digest in the step to pull the image by `docker pull example.contoso.io/app@<digest>`.

Because of the above verification workflow, we need to make sure the output of `notation verify` is script friendly.

The text
```text
Successfully verified for <registry>/<repository>@<digest>
```
is not too bad to be extracted using `sed` or regex.

Reply:
This output looks user-friendly. What's the requirement for a script-friendly output? Keep digest only?

Reply:
For the script-friendly output, `notation` has the same issue as https://github.com/oras-project/oras/issues/638

Reply:
An issue #455  was created for implementing script-friendly output in later release